lib/services/blob/blobservice.core.js

```diff
@@ -5752,7 +5810,7 @@ BlobService.prototype._getBlobToStream = function (container, blob, writeStream,
 *                                                                 the entries of blobs and the continuation token for the next listing operation.
 *                                                                 `response` will contain information related to this operation.
 */
-BlobService.prototype._listBlobsOrDircotriesSegmentedWithPrefix = function (container, prefix, currentToken, listBlobType, optionsOrCallback, callback) {
+BlobService.prototype._listBlobsOrDirectoriesSegmentedWithPrefix = function (container, prefix, currentToken, listBlobType, optionsOrCallback, callback) {
```

Comment:
Ok, consider the backward compatibility of method with underline make sense, as it's not prevented being used by existing user,  please add comment for TODO: renaming, and we will rename the method when there is major version. At same time I'm ok for the solution, keeping entries only for backward compatibility consideration of method _listBlobsOrDircotriesSegmentedWithPrefix, and please add comment for TODO: removing the entries in _listBlobsOrDircotriesSegmentedWithPrefix.

As you know, both keeping fileEntries, directoryEntries and entries in same method could be ambiguous, so we can remove entries, and use Option3+mapping, in the future breaking change.